app/views/layouts/overdue.html.erb

```diff
@@ -1,118 +1,70 @@
 <% content_for(:content) do %>
-  <!---------------------
-  MOBILE NAV
-  --------------------->
-  <div class="mobile">
-    <div class="mt-0 mb-4 active-filters-mobile">
-      <a class="filters-modal-toggle" data-toggle="modal" data-target="#filter-modal">
-          <span class="filter-tags">
-            <% (Array(@search_filters)).each do |filter| %>
-              <% label = PatientSummaryQuery.label_for(filter) %>
-              <span class="badge badge-pill badge-secondary"><%= label %></span>
-            <% end %>
-          </span>
-        <i class="fas fa-filter"></i> Search filters
-      </a>
-    </div>
-
-    <section class="mb-4">
-      <% if current_facility %>
+  <h1>
+    Overdue patients
+  </h1>
+  <p>
+    Patients that are overdue for a follow-up visit. If you select a facility, you can download a patient list.
+  </p>
+  <% if current_admin.accessible_facilities(:manage_overdue_list) %>
+    <%= render('shared/paginate_and_filter_by_facility',
+               form_url: appointments_path,
+               current_admin: current_admin,
+               facility_id: @facility_id,
+               per_page: @per_page,
+               form_id: "search-filters",
+               scope_namespace: current_admin.accessible_facilities(:manage_overdue_list)) %>
+  <% else %>
+    <%= render('shared/paginate_and_filter_by_facility',
+               form_url: appointments_path,
+               current_admin: current_admin,
+               facility_id: @facility_id,
+               per_page: @per_page,
+               form_id: "search-filters",
+               scope_namespace: [:overdue_list, Facility]) %>
+  <% end %>
+  <section class="secondary-nav-filters">
+    <h4 class="mt-2">
+      Filters
+    </h4>
+    <%= form_tag appointments_path, method: :get, id: "search-filters" do %>
+      <%= hidden_field_tag :submitted, true %>
+      <% PatientSummaryQuery.filters.each do |filter, label| %>
+        <label for=<%= "search-#{filter}" %>>
+          <% checked = @search_filters.include?(filter) %>
+          <%= check_box_tag("search_filters[]", filter, checked, { id: "search-#{filter}" } ) %>
+          <%= label %>
+        </label>
+      <% end %>
+      <label>
+        <button type="submit" class="btn btn-sm btn-outline-primary">
+          Apply filters
+        </button>
+      </label>
+    <% end %>
+  </section>
+  <section class="mb-5">
+    <% if current_facility %>
+      <% if current_admin.permissions_v2_enabled? %>
```

Comment:
This is causing a spec failure ^. The `permissions_v2_enabled?` method doesn't exist anymore